lib/active_model/serializer/attributes.rb

```diff
@@ -14,10 +14,12 @@ module Attributes
 
         # Return the +attributes+ of +object+ as presented
         # by the serializer.
-        def attributes(requested_attrs = nil, reload = false)
+        def attributes(options = {}, reload = false)
+          requested_attrs = options[:only]
+          excepts = Array(options[:except])
```

Comment:
I mean, I like the idea of accepting requested options and excluded options, but this could be done, with a little more effort, via attribute `unless` and `instance_options`.  

The PR is great.  I'm just uncertain if I want it as a new feature to maintain, or if it's a first class behavior... Perhaps some other @rails-api/ams @rails-api/commit maintainers could chime in